include/common.h

```diff
@@ -73,6 +73,7 @@ struct nvnc_client {
 	int n_pending_requests;
 	bool is_updating;
 	nvnc_client_fn cleanup_fn;
+	nvnc_client_cut_fn client_cut_fn;
```

Comment:
What scenario do you have in mind where it's useful to be able to set a per-client cut text callback?